app/src/main/java/com/hedvig/app/ApplicationModule.kt

```diff
@@ -1,3 +1,5 @@
+@file:Suppress("RemoveExplicitTypeArguments")
```

Comment:
I *really* like being able to specify the explicit type in the DI configuration and not have the IDE yell at me for being explicit.
I enjoy being able to glance over declarations where it can look like:
```kotlin
single<Type> { //my eyes often don't need to go here  }
single<Type> { ... }
single<Type> { ... }
single<Type> { ... }
```

compared to

```kotlin
```kotlin
single { SomeType(get()) } // simple case
single { (someParameter) -> SomeType(someParameter) } // a bit more involved
single { (someParameter: ExplicitTypeDeclaration) -> SomeType(someParameter) } // The type is hidden even further to the right
single { (someParameter: ExplicitTypeDeclaration, someParameter2: ExplicitTypeDeclaration) -> SomeType(someParameter, someParameter2) } // a bit more
single { (someInputParameter: ExplicitTypeDeclaration ->
    val interfacesList = getAll<SomeInterface>().distinct().toList()
    TheTypeImInterestedIn(interfacesList, someInputParameter) 
} // When there's even more logic involved, becomes even harder again to at a glance look at it
```